apps/dev/pages/api/auth/[...nextauth].ts

```diff
@@ -193,6 +194,12 @@ export const authOptions: NextAuthOptions = {
       clientId: process.env.PATREON_ID,
       clientSecret: process.env.PATREON_SECRET,
     }),
+    AsgardeoProvider({
+      organization: process.env.ASGARDEO_ORGANIZATION,
+      clientId: process.env.ASGARDEO_CLIENT_ID,
+      clientSecret: process.env.ASGARDEO_CLIENT_SECRET,
+      scopes: process.env.ASGARDEO_SCOPES,
```

Comment:
```suggestion
```

I don't think we should introduce new options, it would be confusing since other providers don't have this.